WordPress/gutenberg
# Add characters count in the table of contents

## Description
fix #13796 

> When writing a post in Chinese the word count shown in the content structure does not show an accurate word count.

## Screenshot
![image](https://user-images.githubusercontent.com/24849874/54852493-65194780-4cc3-11e9-8af2-7195ff820b8f.png)


## How has this been tested?
- [X] Browser testing

## Types of changes
Add the count of characters(no spaces) in the table of contents

## Testing

Steps to reproduce the behavior:
1. Use http://generator.lorem-ipsum.info/_chinese to generate some sample Chinese text
2. Create new page in Gutenberg
3. Paste in content
4. Click the info icon at the top
5. See if the number of characters is correct

## Checklist:
